example/Cargo.toml

```diff
@@ -14,6 +14,4 @@ console_error_panic_hook = "0.1.6"
 
 [dependencies.web-sys]
 version = "0.3.35"
-features = [
-    "console",
-]
+features = ["console"]
```

Comment:
I'd prefer to keep the old formatting, because it makes it easier to add new features without causing git diff hell.